tests/test_utilsEfd.py

```diff
@@ -29,12 +29,13 @@
 class UtilsEfdTestCase(lsst.utils.tests.TestCase):
     """Unit test for EFD access code."""
 
-    def setUp(self):
+    @classmethod
+    def setUpClass(cls):
         # Initialize a client.
         try:
-            self.client = CpEfdClient()
+            cls.client = CpEfdClient()
         except Exception as e:
```

Comment:
Is this still supposed to give up on any Exception? It appears as if it should run if you can connect to the EFD, no?

Reply:
If it can't connect to the EFD with any error it should skip the tests.  I'm not sure what your question is here.

Reply:
My question would be: why do this at all, and not just use `vcr` to test the code exactly the same way, but with no dependency on the EFD at all? That way the tests always do run, and you don't have to ever worry about connections.

Reply:
All I meant was that you are raising a `RuntimeError` in `checkConnection` but here you catch any exception from `CpEfdClient.__init__`. But I suppose that's fine since it is reraising and not just continuing.

Reply:
Would you prefer that this just check for a RuntimeError?

Reply:
Eh, never mind. Maybe reconsider if/when you switch to vcr and maybe it'll just be moot.